ecommerce/models.py

```diff
@@ -9,6 +9,14 @@
 from mitxpro.utils import serialize_model_object
 
 
+class Company(TimestampedModel):
+    """
+    A company that purchases bulk seats/coupons
+    """
+
+    name = models.CharField(max_length=512, unique=True)
```

Comment:
Can you add a simple `__str__` function here?

Reply:
👍